docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -105,24 +157,33 @@ Instead, set up limited demo accounts, or fake credit cards with limited functio
 ====
 
 Use environment variables to handle any values needing encryption at rest. 
```

Comment:
@paulb-elastic 
I worry this isn't entirely true when factoring in private locations. What do you think?

Reply:
This isn't a new change, but it's a good point you make.  We have a [placeholder](https://github.com/elastic/synthetics-dev/issues/207) to improve this in the future when there's more capability in Agent, but that's not there yet.

How about changing it to something like:
> Environment variables can also be used as shown in the example below which uses the syntax ${URL} to reference a variable named URL defined in the environment:

Reply:
>the example below which uses the syntax `${URL}` to reference a variable named `URL`

I'm confused about the syntax to use in the example. The existing example in this doc is a YAML browser monitor, which isn't supported anymore (right?):

```yml
- name: Todos
  id: todos
  type: browser
  schedule: "@every 3m"
  tags: todos-app
  params:
    url: ${MY_URL}
  source:
    inline:
      script: |-
      step("load homepage", async () => {
        await page.goto(params.url);
      });
      step("hover over products menu", async () => {
        await page.hover('css=[data-nav-item=products]');
      });
```

I made up a new example using an env variable in `synthetics.config.ts`:

```js
export default {
  params: {
    my_url: process.env.MY_URL
  }
};
```

Is this right? Should we be showing something different?

Reply:
@colleenmcginnis yes - great point - we've stopped supporting this (browser as inline yml)